src/Atributes/Elements/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Button({terms, callFunction, children}) {
+
+    return (
+        <button
+            type="button"
+            disabled={terms}
+            onClick={callFunction}>
+            {children}
+        </button>
+
+    )
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct <button /> kunnen gebruiken